messages/views.py

```diff
@@ -184,13 +183,12 @@ def new_message(request, username=None, message_id=None):
 def username_lookup(request):
     results = []
     if request.method == "GET":
-                # Only autocompleting for previously contacted users
-                previously_contacted_user = list(Message.objects.filter(user_from = request.user.id)\
-                        .values_list('user_to__username', flat='True').distinct())
-                previously_contacted_user2 = list(Message.objects.filter(user_to = request.user.id)\
-                        .values_list('user_from__username', flat='True').distinct())
+        # Only autocompleting for previously contacted users
+        previously_contacted_user = list(
+            Message.objects.filter(user_from=request.user.id).values_list('user_to__username', flat='True').distinct())
+        previously_contacted_user2 = list(
+            Message.objects.filter(user_to=request.user.id).values_list('user_from__username', flat='True').distinct())
 
-                results = list(set(previously_contacted_user + previously_contacted_user2))
+        results = list(set(previously_contacted_user + previously_contacted_user2))
```

Comment:
I know you're only reformatting this code, but does it make sense for this to return the value of a set? This means that the order of these items could change. It'd be nice to see the list ordered by the frequency of contact.
It would also be nice to make only 1 query (to minimise load on the database)

Reply:
The order is not relevant because this list is used for the JS autocomplete feature, which will further filter the list and show results according to the characters that user is typing in the username box. I nevertheless updated the query so now we only make one. Addressed in bec9e953de2d51b12553146736fcf5a36be14874